llvm/lib/Transforms/IPO/AttributorAttributes.cpp

```diff
@@ -5500,7 +5504,34 @@ struct AAAlignCallSiteReturned final
   using Base = AACalleeToCallSite<AAAlign, AAAlignImpl>;
   AAAlignCallSiteReturned(const IRPosition &IRP, Attributor &A)
       : Base(IRP, A) {}
+  ChangeStatus updateImpl(Attributor &A) override {
+    SmallVector<AA::ValueAndContext> Values;
+    const auto &AligmentCBs = A.getAlignmentCallback(getIRPosition());
+
+    if (!AligmentCBs.empty()) {
```

Comment:
since `lookup` will do a default construct, you don't need to check emptiness here.